cmd/commands/cmd_import_mission_control.go

```diff
@@ -98,3 +101,161 @@ func importMissionControl(ctx *cli.Context) error {
 	_, err = client.XImportMissionControl(rpcCtx, req)
 	return err
 }
+
+var loadMissionControlCommand = cli.Command{
+	Name:     "loadmc",
+	Category: "Payments",
+	Usage: "Load mission control results to the internal mission " +
+		"control state from a file produced by querymc with the " +
+		"option to shift timestamps. Note that this data is not " +
+		"persisted across restarts.",
+	Action: actionDecorator(loadMissionControl),
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "mcdatapath",
+			Usage: "The path to the querymc output file (json).",
+		},
+		cli.BoolFlag{
+			Name:  "discard",
+			Usage: "Discards current mission control data.",
+		},
+		cli.StringFlag{
+			Name: "timeoffset",
+			Usage: "Time offset to add to all timestamps. " +
+				"Format: 1m for 1 minute, 1h for 1 hour, 1d " +
+				"for 1 day.",
+		},
+	},
+}
+
+// loadMissionControl loads mission control data into an LND instance.
+func loadMissionControl(ctx *cli.Context) error {
+	rpcCtx := context.Background()
+
+	mcDataPath := ctx.String("mcdatapath")
+	if mcDataPath == "" {
+		return fmt.Errorf("mcdatapath must be set")
+	}
+
+	if _, err := os.Stat(mcDataPath); os.IsNotExist(err) {
+		return fmt.Errorf("%v does not exist", mcDataPath)
+	}
+
+	conn := getClientConn(ctx, false)
+	defer conn.Close()
+
+	client := routerrpc.NewRouterClient(conn)
+
+	// Load and unmarshal the querymc output file.
+	mcRaw, err := os.ReadFile(mcDataPath)
+	if err != nil {
+		return fmt.Errorf("could not read querymc output file: %w", err)
+	}
+
+	mc := &routerrpc.QueryMissionControlResponse{}
+	err = lnrpc.ProtoJSONUnmarshalOpts.Unmarshal(mcRaw, mc)
+	if err != nil {
+		return fmt.Errorf("could not unmarshal querymc output file: %w",
+			err)
+	}
+
+	// We discard mission control data if requested.
+	if ctx.Bool("discard") {
+		if !promptForConfirmation("This will discard mission control " +
```

Comment:
nit: add word "current" here as well: `This will discard all current mission control data in the database: (yes/no)`.

Reply:
makes sense, done!